library/Garp/Cache/Manager.php

```diff
@@ -155,6 +157,47 @@ public static function purgeStaticCache($modelNames = array(), $cacheDir = false
     }
 
     /**
+     * This clears the OpCache. This reset can only be done with an http request.
+     *
+     * @return Void
+     */
+    public static function purgeOpCache() {
+        // get deployment file
+        if (APPLICATION_ENV === 'development' || APPLICATION_ENV === 'testing') {
```

Comment:
Or perhaps even better: simply detect wether opcache is running?

Reply:
+1

Reply:
I corrected the env-dependent part, replacing it for a check for Opcache existence.
Plus I added an Opcache clear in CLI context, since these are often separated.